docs/using_lagoon/lagoon_yml.md

```diff
@@ -83,6 +88,10 @@ This allows you to define the behaviour of the automatic creates routes (NOT the
 
 There are different type of tasks you can define, they differ when exactly they are executed in a build flow:
 
+### `pre_rollout.[i].run`
+The taks defined as pre_rollout tasks will run against your project _before_ the new Images get built.
```

Comment:
ahem no? 
please see
https://github.com/amazeeio/lagoon/blob/master/images/oc-build-deploy-dind/build-deploy-docker-compose.sh#L155
and 
https://github.com/amazeeio/lagoon/blob/master/images/oc-build-deploy-dind/build-deploy-docker-compose.sh#L180

it's after the images are built (purposely so that the developers know that their docker images can be built), but before anything is changed on the kubernetes objects are changed (aka the rollout).

please fix :)